cpp/include/cudf/io/datasource.hpp

```diff
@@ -86,28 +86,24 @@ class datasource {
   /**
    * @brief Creates a source from a file path.
    *
-   * @note Parameters `offset`, `max_size_estimate` and `min_size_estimate` are hints to the
-   * `datasource` implementation about the expected range of the data that will be read. The
-   * implementation may use these hints to optimize the read operation. These parameters are usually
-   * based on the byte range option. In this case, `min_size_estimate` should be no greater than the
-   * byte range to avoid potential issues when reading adjacent ranges. `max_size_estimate` can
-   * include padding after the byte range, to include additional data that may be needed for
-   * processing.
+   * @note Parameters `offset` and `max_size_estimate` are hints to the `datasource`
```

Comment:
```suggestion
   * Parameters `offset` and `max_size_estimate` are hints to the `datasource`
```
I don't think `@note` is necessary here.

Reply:
Thanks! Done. 
Also the comment section has been reformatted.